it/test/controllers/manageBusinesses/manage/ManageIncomeSourceDetailsUKPropertyControllerISpec.scala

```diff
@@ -63,7 +62,8 @@ class ManageIncomeSourceDetailsUKPropertyControllerISpec extends ManageIncomeSou
                 elementTextBySelectorList("#manage-details-table", "div:nth-of-type(1)", "dt")("Date started"),
                 elementTextBySelectorList("#manage-details-table", "div:nth-of-type(1)", "dd")(businessStartDate),
                 elementTextBySelectorList("#manage-details-table", "div:nth-of-type(2)", "dt")("Accounting method"),
-                elementTextBySelectorList("#manage-details-table", "div:nth-of-type(2)", "dd")(businessAccountingMethod)
+                elementTextBySelectorList("#manage-details-table", "div:nth-of-type(2)", "dd")(businessAccountingMethod),
+                elementTextByID("up-to-two-tax-years")("")
```

Comment:
if I understand this correctly this test is for the scenario for no latency information returned would we want a test for when there is latency information returned and the correct content is returned?